docker-compose.override.yaml

```diff
@@ -0,0 +1,15 @@
+version: '3'

```

Comment:
override and local always seems to be used together, could they be combined?

Reply:
The default docker-compose and override provide everything needed to run this service in isolation for local testing (i.e. no dependence on the external `mine-support` network). This means you can run locally without creating the external network using just `docker-compose build/up/down` with no `-f` arguments.

The local override adds the external Docker network needed to connect this service with others running locally.